Mathlib/MeasureTheory/Function/ConvergenceInMeasure.lean

```diff
@@ -62,6 +66,49 @@ theorem tendstoInMeasure_iff_norm [SeminormedAddCommGroup E] {l : Filter ι} {f
   simp_rw [TendstoInMeasure, dist_eq_norm]
 #align measure_theory.tendsto_in_measure_iff_norm MeasureTheory.tendstoInMeasure_iff_norm
 
+-- The ae-limit is ae-unique.
+theorem tendsto_ae_unique {α ι E : Type*} [TopologicalSpace E] [T2Space E]
```

Comment:
Can you see if you can move this to either where `tendsto_nhds_unique` is or where the `ae` filter is defined?

Reply:
I agree that it is/was not in a good place. I think `MeasureTheory.Measure.Function.AEEqFun.lean` is a good place and I will put it there.